src/main/resources/db/migration/V1_38__document_simpler_tables.sql

```diff
@@ -0,0 +1,29 @@
+COMMENT ON TABLE auth_user IS 'details about the user from hmpps-auth';
+COMMENT ON COLUMN auth_user.id IS 'the user ID';
+COMMENT ON COLUMN auth_user.auth_source IS 'where the user has come from';
+COMMENT ON COLUMN auth_user.user_name IS 'the username';
+
+COMMENT ON TABLE service_category IS '[reference data] intervention service categories, which relate to service user needs';
+COMMENT ON COLUMN service_category.id IS 'service-owned unique identifier';
+COMMENT ON COLUMN service_category.created IS 'when the record was added';
+COMMENT ON COLUMN service_category.name IS 'intervention service category';
+
+COMMENT ON TABLE complexity_level IS '[reference data] complexity levels for each intervention service category';
```

Comment:
TIL this appears as a link in the app -- so I suppose we can use Markdown here
![image](https://user-images.githubusercontent.com/1526295/113689886-b3d78580-96c2-11eb-9e9f-f530ba8642d9.png)